src/test/javascript/spec/component/exam/participate/summary/exam-participation-summary.component.spec.ts

```diff
@@ -174,4 +188,103 @@ describe('ExamParticipationSummaryComponent', () => {
         expect(printStub).toHaveBeenCalled();
         printStub.mockRestore();
     }));
+
+    it('should correctly identify a TestExam', () => {
+        studentExam.exam = TestExam;
+        component.studentExam = studentExam;
+        component.ngOnInit();
+        expect(component.isTestExam).toBeTrue;
+        expect(component.testExamConduction).toBeTrue;
+
+        studentExam.submitted = true;
+        component.studentExam = studentExam;
+        component.ngOnInit();
+        expect(component.isTestExam).toBeTrue;
+        expect(component.testExamConduction).toBeFalse;
```

Comment:
Seems like you forgot this everywhere in this file, please check below as well
```suggestion
        expect(component.isTestExam).toBeTrue();
        expect(component.testExamConduction).toBeTrue();

        studentExam.submitted = true;
        component.studentExam = studentExam;
        component.ngOnInit();
        expect(component.isTestExam).toBeTrue();
        expect(component.testExamConduction).toBeFalse();
```